nntrainer/layers/fc_layer.cpp

```diff
@@ -186,9 +187,11 @@ void FullyConnectedLayer::forwarding(RunLayerContext &context, bool training) {
     Tensor &loraB = context.getWeight(lora_idx[LORAParams::loraB]);
     Tensor &hidden_tmp_lora = context.getTensor(lora_idx[LORAParams::loraTmp]);
     Tensor &hidden_out_lora = context.getTensor(lora_idx[LORAParams::loraOut]);
+    auto lora_scaling = std::get<props::LoraScaling>(fc_props).get();
```

Comment:
could we use `const auto &` to prevent coverity issue
```suggestion
    const auto &lora_scaling = std::get<props::LoraScaling>(fc_props).get();
```